README.md

```diff
@@ -1,53 +1,55 @@
 
-Development: for devs who want to dig into the code, see our hacking [README](hack/README.md#getting-started-for-developers) (WIP)
 
 ## Purpose
 
-This project eases the burden on cluster admins seeking to take advantage of Kubernetes/Openshift orchestration of their virtualized app platforms. For the purposes of running a VM inside a container, the imported file referred to above is a VM image and is considered to be an immutable _golden image_  for subsequent cloning and instantiation. As a first step in migration to a Kubernetes cluster, virtual machine images must be imported into a location accessible to the kubelet. The importer automates this by copying images from an external http repository and persisting them in in-cluster storage. The components of this process are detailed [here](/doc/design.md#design).
+This project is designed with Kubevirt in mind and provides a declarative method of importing VM images into a Kuberenetes cluster. Through this behavior, cluster administrators can build an abstract registry of template images (referred to as "Golden Images" for their role as templates for image clones).
+
+For an in depth look at the system, see the [Design](/doc/design.md#design) documentation.
 
 ### Data Formatting
 
-While the importer has been designed to be relatively agnostic to file format, it is capable of performing certain functions that streamline its use with Kubevirt.  It will unpackage **gzip** and **tar** archived files automatically, as well as convert **qcow2** images into raw image files.  
+The importer is capable of performing certain functions that streamline its use with Kubevirt.  It will unpackage **gzip** and **tar** archived files automatically, as well as convert **qcow2** images into raw image files.
 
-## CDI Tire Kicking
+Expected file formats are:
 
-Deploying the controller pod is simple!
+- .tar
+- .gz
+- .img
+- .iso
+- .qcow2
 
-1. Secret(s) containing endpoint credentials. Not required if the endpoint(s) are public.
-1. Storage class(es) defining the backend storage provisioner(s).
-1. Controller pod via the Deployment template. Note that the controller pod spec needs to set an environment variable named OWN_NAMESPACE which can be done as:
-```
-...
-   imagePullPolicy: Always
-       env:
-         - name: OWN_NAMESPACE
-           valueFrom:
-             fieldRef:
-               fieldPath: metadata.namespace
-```
+## Deploying CDI
 
 ### Assumptions
 - A running Kubernetes cluster
-- A reachable object store
-- A file in the object store to be imported.
+- A storage class and provisioner.
+- An HTTP or S3 file server hosting VM images
+- A namespace acting as the image registry (`default` is fine for tire kicking)
+
+Deploying the CDI controller is straight forward.  Create the controller in the namespace where VM images are to be stored.
+
+`$ kubectl create -f https://raw.githubusercontent.com/kubevirt/containerized-data-importer/master/manifests/cdi-controller-deployment.yaml`
```

Comment:
This manifest does not seem to name the cdi-controller pod. The image, deployment, and labels are named but not the name of the controller pod, AFAICT.

Reply:
Right, deployments will name the pods after the deployment, so the controller pod appears as `cdi-deployment-vxyz`.